.github/workflows/build-bootc.yaml

```diff
@@ -32,121 +35,47 @@ jobs:
+          file: bootc-agent-images/${{ matrix.flavor }}/Containerfile
+          load: true
+          tags: user/flightctl-agent:test
 
-      - name: Modify RHEL Containerfile
+      - name: Test image
         run: |
-          pushd bootc-agent-images/rhel
-          echo "${{ secrets.FLIGHTCTL_RSA_PUB }}" > flightctl_rsa.pub
-          sed -i -e 's/^# COPY flightctl_rsa.pub \/usr\/etc-system\/root.keys/COPY flightctl_rsa.pub \/usr\/etc-system\/root.keys/' \
-                 -e 's/^# RUN touch \/etc\/ssh\/sshd_config.d\/30-auth-system.conf;/RUN touch \/etc\/ssh\/sshd_config.d\/30-auth-system.conf;/' \
-                 -e 's/^#     mkdir -p \/usr\/etc-system\/;/    mkdir -p \/usr\/etc-system\/;/' \
-                 -e 's/^#     echo '\''AuthorizedKeysFile \/usr\/etc-system\/%u.keys'\'' >> \/etc\/ssh\/sshd_config.d\/30-auth-system.conf;/    echo '\''AuthorizedKeysFile \/usr\/etc-system\/%u.keys'\'' >> \/etc\/ssh\/sshd_config.d\/30-auth-system.conf;/' \
-                 -e 's/^#     chmod 0600 \/usr\/etc-system\/root.keys/    chmod 0600 \/usr\/etc-system\/root.keys/' \
-                 -e 's/^# VOLUME \/var\/roothome/VOLUME \/var\/roothome/' Containerfile
-          echo "${{ secrets.CA_CRT }}" > ca.crt
-          echo "${{ secrets.CLIENT_ENROLLMENT_CRT }}" > client-enrollment.crt
-          echo "${{ secrets.CLIENT_ENROLLMENT_KEY }}" > client-enrollment.key
-          popd
-
-      - name: Set up QEMU
-        uses: docker/setup-qemu-action@v1
-
-      - name: Login to Red Hat Registry
-        uses: docker/login-action@v3
-        with:
-          registry: registry.redhat.io
-          username: ${{ secrets.RH_REGISTRY_USERNAME }}
-          password: ${{ secrets.RH_REGISTRY_PASSWORD }}
-
-      - name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@v1
-
-      - name: Login to Quay.io
-        uses: docker/login-action@v3
-        with:
-          registry: ${{ env.REGISTRY }}
-          username: ${{ secrets.QUAY_FLIGHTCTL_INFRA_ROBOT_USERNAME }}
-          password: ${{ secrets.QUAY_FLIGHTCTL_INFRA_ROBOT_PASSWORD }}
+          ID=$(docker create ${{ steps.build.outputs.imageid }} /verify-flightctl-agent.sh)
+          docker cp .github/workflow-scripts/verify-flightctl-agent.sh $ID:/
+          docker start $ID
+          exit $(docker inspect $ID --format='{{.State.ExitCode}}')
 
-      - name: Build and push
+      - name: Push image
         uses: docker/build-push-action@v2
         with:
-          context: bootc-agent-images/rhel
-          file: bootc-agent-images/rhel/Containerfile
+          context: bootc-agent-images/${{ matrix.flavor }}
+          file: bootc-agent-images/${{ matrix.flavor }}/Containerfile
           platforms: linux/amd64,linux/arm64
           push: true
-          tags: ${{ env.REGISTRY }}/${{ env.REPOSITORY }}/flightctl-agent-rhel:bootstrap
+          tags: ${{ env.REGISTRY }}/${{ secrets.QUAY_FLIGHTCTL_INFRA_ROBOT_USERNAME }}/flightctl-agent-${{ matrix.flavor }}:bootstrap
```

Comment:
for bootstrap images, we have to use flightctl.

Look at the org hosting the flightctl-agent-{variant} repos: https://quay.io/organization/flightctl